tests/benchmark/bm_datasets.py

```diff
@@ -29,7 +29,8 @@ def get_data_set(dataset_name):
 
 
 # Create an HNSW index from dataset based on specific params.
-def create_hnsw_index(dataset, ef_construction, M):
+def load_or_create_hnsw_index(dataset, index_file_name, ef_construction, M):
```

Comment:
Did you ran this script locally to see that these changes work?

Reply:
yes i ran make benchmark